tests/testthat/_snaps/discretize.md

```diff
@@ -73,7 +74,7 @@
     Condition
       Error in `step_discretize()`:
       Caused by error in `recipes::discretize()`:
-      ! There should be at least 2 cuts
+      ! `cuts` must be at least 2, not 1.
```

Comment:
I think I prefer something closer to the old wording. How about:

```suggestion
      ! There should be at least 2 `cuts`, but 1 was supplied.
```

Reply:
done in https://github.com/tidymodels/recipes/pull/1255/commits/483af8a84507657d909bec152f8bb87e15fb9350